project1/backend/models/resolutionModel.js

```diff
@@ -0,0 +1,86 @@
+const AppError = require('../utils/AppError');
+const formatName = require('../utils/formatName.js');
+const Ajv = require('ajv');
+
+const ajv = new Ajv();
+
+class ResolutionModel {
```

Comment:
Probably, here should be relation between QueueModel and ResolutionModel?

Reply:
No, these are separate, the relation was required previously because we needed data of the client from the Queue when we create a new Resolution data entry, but this we simply send a request to the queue endpoint regarding the current customer.